src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,110 @@
+    >
+      <div>
+        <GapChart
+          :active-stats="activeStats"
+          :expected="expected"
+          :field-data="fieldData"
+        />
+      </div>
+      <div class="centered">
+        <p class="has-text-centered">
+          In {{ activeCluster.name }}, the largest gap is among
+          {{ activeStats[0]?.name }} residents. Only
+          <strong
+            >{{ formatPct(activeStats[0]?.pct) }} of
+            {{ activeStats[0]?.name }} residents</strong
+          >
+          are vaccinated compared to {{ formatPct(expected) }} statewide.
+          <strong
+            >{{ activeStats[0]?.gap }} more
+            {{ activeStats[0]?.name }} residents</strong
+          >
+          need to be vaccinated to close this gap.
+        </p>
+      </div>
     </div>
-    <div class="centered">
-      <p class="has-text-centered">
-        In {{ activeCluster.name }}, the largest gap is among
-        {{ minVaxRace?.name }} residents. Only
-        <strong
-          >{{ formatPct(minVaxRace?.pct) }} of
-          {{ minVaxRace?.name }} residents</strong
-        >
-        are vaccinated compared to {{ formatPct(expected) }} statewide.
-        <strong
-          >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
-        >
-        need to be vaccinated to close this gap.
-      </p>
+
+    <!-- A focus stat selected, display KPI -->
+    <div
+      class="gap-container"
+      :class="{ visible: focusStat.value !== 'total' }"
+    >
+      <div
+        class="is-flex is-flex-direction-row is-justify-content-space-around"
+      >
+        <KeyPerformanceIndicator
+          :value="
+            activeFocusStats?.population > 0
+              ? formatPct(activeFocusStats?.pct)
+              : '?'
+          "
+          :title="`${activeFocusStats?.name} residents vaccinated in ${activeCluster.name}`"
+        />
+        <KeyPerformanceIndicator
+          :value="
+            activeFocusStats?.population > 0 ? activeFocusStats?.gap : '?'
+          "
+          :title="`Vaccine doses for ${activeFocusStats?.name} residents needed to close the gap`"
+        />
+      </div>
+      <div class="content centered">
+        <!-- In this community, there is a gap in this focus group -->
+        <p v-if="activeFocusStats?.gap > 0" class="has-text-centered">
+          In {{ activeCluster.name }},
+          <strong>{{ formatPct(activeFocusStats?.pct) }}</strong> of
+          {{ activeFocusStats?.name }} residents are vaccinated compared to our
+          goal of {{ formatPct(expected) }} total vaccinations statewide.
+          <strong
+            >{{ activeFocusStats?.gap }} more
+            {{ activeFocusStats?.name }} residents</strong
+          >
+          need to be vaccinated to close this gap.
+        </p>
+        <!-- There is no gap in this focus group, display the largest gap -->
+        <span v-else class="has-text-centered">
```

Comment:
should this class be moved to the outer container?

Reply:
Yes it can be! I've moved it